samples/nrf9160/secure_services/Kconfig

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+source "Kconfig.zephyr"
+
+menu "Secure Service Sample"
+
+config SAMPLE_SERVICE_RNG
+	bool "Enable code that calls the spm_request_random_number()"
+	default y
+
+# NORDIC_SECURITY_BACKEND is not supported on 53 yet
+	depends on ! SOC_NRF5340_CPUAPP
```

Comment:
seems wrong to have this dependency, is there nothing else you can depend upon, like `NORDIC_SECURITY_BACKEND` or `cc3xx` or ?
So that when it is supported on nRF53, then this code will automatically be active.

Reply:
Makes sense. @tejlmand or @frkv please tell me if there is a symbol this can depend on that is better than SOC_NRF5340_CPUAPP.

Alternatively, we can change it in a separate PR.

Reply:
Yes. This should point to the CC310 entropy driver, which has a config. In time it will point to CC312 entropy driver as well, but this would need to go in in a different PR

It won't build for nRF5340 ATM.